include/rppt_tensor_effects_augmentations.h

```diff
@@ -28,6 +28,7 @@ THE SOFTWARE.
 extern "C" {
 #endif
 
+RppStatus rppt_glitch_host(RppPtr_t srcPtr, RpptDescPtr srcDescPtr, RppPtr_t dstPtr, RpptDescPtr dstDescPtr, Rpp32u *x_offset_r, Rpp32u *y_offset_r, Rpp32u *x_offset_g, Rpp32u *y_offset_g, Rpp32u *x_offset_b, Rpp32u *y_offset_b, RpptROIPtr roiTensorPtrSrc, RpptRoiType roiType, rppHandle_t rppHandle);
```

Comment:
Please add this declaration at the end, not at the beginning at the description for the params similar to the other function declarations

Since all the params are using only camel case it is better to stick to that, Please use camel case for offset variables like xOffsetR, yOffsetR